tests/test_main.py

```diff
@@ -31,13 +28,48 @@ def test_help(arg, capsys):
 
 @mark.parametrize("arg", ("--version", "-v"))
 def test_version(arg, capsys):
-    run_main(arg, code=0)
+    run_main(arg, expected_exit_code=0)
     stdout, _ = capsys.readouterr()
     assert "retasc" in stdout
 
 
 def test_dummy_run(capsys):
-    run_main()
+    run_main(expected_exit_code=0)
     stdout, stderr = capsys.readouterr()
     assert stdout == ""
     assert stderr == ""
+
+
+@patch("retasc.__main__.validate_rule")
+def test_validate_rule(mock_validate_rule):
+    run_main("validate-rule", "any_valid_rule.yaml", expected_exit_code=0)
+    mock_validate_rule.assert_called_once_with("any_valid_rule.yaml")
```

Comment:
Better to drop this. It is already tested in `test_validate_rule_output` ... thought this only checks that the validation function is called which does not verify any useful behavior.

```suggestion
```